CONTRIBUTING.md

```diff
@@ -2,48 +2,68 @@
 
 This document contains information for anyone wishing to contribute to the project.
 
-## Install
+
+
+## Installation
 
 The repo includes a simple test application that can be run locally to develop the pattern library itself.
+
 First, clone the repo:
 
 ```sh
 git clone git@github.com:torchbox/django-pattern-library.git
 cd django-pattern-library
 ```
 
-### Run a local build with Poetry
+Once you have the code, there are several ways of running the project:
 
-We use [Poetry](https://poetry.eustace.io/docs/) to manage Python dependencies, so make sure you've got it installed.
+- [In a VS Code devcontainer](#vs-code-devcontainer)
+- [In Docker, via docker-compose](#docker-compose)
+- [Locally, with Poetry](#run-locally-with-poetry)
 
-Then you can install the dependencies and run the test app:
+### VS Code devcontainer
 
-```sh
-poetry install
-# Start the server for testing:
-poetry run django-admin runserver --settings=tests.settings.dev --pythonpath=.
-# Or to try out the render_patterns command:
-poetry run django-admin render_patterns --settings=tests.settings.dev --pythonpath=. --dry-run --verbosity 2
-```
+For users of Docker and VS Code, there is a [devcontainer](https://code.visualstudio.com/docs/remote/containers) setup included in the repository
+that will automatically install the Python dependencies and start the frontend tooling.
+
+Once the container is built, open a terminal with VS Code and run `django-admin runserver` and click the URL (normally http://127.0.0.1:8000/) to open the app in your browser. You'll see a 404 page, because there's nothing at `/`, add `pattern-libary/` to the end of the URL to view the demo app.
```

Comment:
Should we change our dev site to have the pattern library at the root?

Reply:
Yes, good idea.